src/frontend/components/archive/ArchiveCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import axios from "axios"
+import { useAuthContext, useHabit } from "../../context/index-context"
```

Comment:
Suggestion-you can just name it as useAuth only , since it is a custom hook, no need to specify context